Update Getting Started guide with extenstion for VS Code (#3639)
vividus-framework/vividus: docs/modules/ROOT/pages/getting-started.adoc

== Use IDE for the tests development

> Please choose any of the following IDEs

Reviewer: ```suggestion
Please choose any of the following IDEs.
```

. https://help.eclipse.org/2022-03/index.jsp?topic=%2Forg.eclipse.platform.doc.user%2Ftasks%2Ftasks-importproject.htm[Import the project to Eclipse]
. Start creating your tests

=== Visual Studio Code

==== Prerequisites

> 

Reviewer: at first you need to install VS Code

> . Install https://marketplace.visualstudio.com/items?itemName=vividus.vividus-studiop[VIVIDUS Studio]

Reviewer 2: Its 404 for the link

Reviewer 2: BTW marketplace uses the following link: 
<img width="385" alt="image" src="https://user-images.githubusercontent.com/17816729/218698840-24b74216-4af3-4625-ba65-b88e9de34fab.png">
Maybe we could also have some sort of shortcut here?

Reviewer: most likely this shortcut won't work if VS code is not installed, or installed, but system is not rebooted

Author: I would prefer to keep the link to marketplace so user can at least read about the extension

> . Install https://marketplace.visualstudio.com/items?itemName=vividus.vividus-studiop[VIVIDUS Studio]

Reviewer: ```suggestion
. Install https://marketplace.visualstudio.com/items?itemName=vividus.vividus-studio[VIVIDUS Studio]
```

> ==== Open the project in Visual Studio Code

Reviewer 2: what about tests execution?

Author: Good idea but I would prefer to put this into extension

Reviewer 2: I think it should be here, despite the absence of this section in eclipse :)

. Open Visual Studio Code
. Click `File` -> `Open...`
. Open a folder with VIVIDUS project

> . Wait until project initialization is done

Reviewer 2: ```suggestion
. Wait until the project initialization is done
. Start creating your tests
```

> . Wait until project initialization is done

Reviewer: how to understand when project initialization is completed?

. https://help.eclipse.org/2022-03/index.jsp?topic=%2Forg.eclipse.platform.doc.user%2Ftasks%2Ftasks-importproject.htm[Import the project to Eclipse]
. Start creating your tests

=== Visual Studio Code

==== Prerequisites

. Install https://code.visualstudio.com[Visual Studio Code]
. Install https://marketplace.visualstudio.com/items?itemName=vividus.vividus-studio[VIVIDUS Studio Extension]

==== Open the project in Visual Studio Code

> . Open Visual Studio Code
> . Click `File` -> `Open...`
> . Open a folder with a VIVIDUS project
> . Wait until the message `Project with the name '...' is loaded` appears
> . Start creating your tests

Reviewer 2: ```suggestion
. Open Visual Studio Code.
. Click `File` -> `Open...`.
. Open a folder with a VIVIDUS project.
. Wait until the message `Project with the name '...' is loaded` appears.
. Start creating your tests.
```

==== Execute created tests

> . Open https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette[Command Palette]

Reviewer: it's better to provide simple steps how to open command palette

> . Type `VIVIDUS: Run Stories` in the search bar and select matched command

Reviewer: and what's next?

Author: isn't it clear from the section title :) what would you like me to add here?

Reviewer: what's happened next? how can users understand the tests run has started successfully?

Author: Do we really need this? the following are few actions from getting started guide:
* Install and configure Git - where is info on how to understand that the git was successfully installed?
* Build the project: - how the users understand that the build is successful?
* Execute the tests via CLI - how the users understand that tests has started successfully?
* ....

. https://help.eclipse.org/2022-03/index.jsp?topic=%2Forg.eclipse.platform.doc.user%2Ftasks%2Ftasks-importproject.htm[Import the project to Eclipse]
. Start creating your tests

=== Visual Studio Code

==== Prerequisites

. Install https://code.visualstudio.com[Visual Studio Code].
. Install https://marketplace.visualstudio.com/items?itemName=vividus.vividus-studio[VIVIDUS Studio Extension].

==== Open the project in Visual Studio Code

. Open Visual Studio Code.
. Click `File` -> `Open...`.
. Open a folder with a VIVIDUS project.
. Wait until the message `Project with the name '...' is loaded` appears.
. Start creating your tests.

> ==== Execute created tests

Reviewer 2: ```suggestion
==== Execute tests
```

> . Open https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette[Command Palette] by clicking `View` -> `Command Palette...`.

Reviewer: does it make sense to use https://docs.asciidoctor.org/asciidoc/latest/macros/ui-macros/#menu-macro-syntax ?

. Type `VIVIDUS: Run Stories` in the search bar and select matched command.

> . Check the tests execution progress in the `Output` console, if it's closed open it by clicking `View` -> `Output`.

Reviewer: ```suggestion
. Find the tests execution progress logged in the `Output` console (if it's closed open it by clicking `View` -> `Output`).
```
